### PR TITLE
Allow seed phrases with a trailing newline

### PR DESCRIPTION
**Pull request by whymarrh**
Refs #6134 and #5827

Take 3 🎬 

This PR updates the import seed phrase screen (during on-boarding) to strip extraneous leading and trailing whitespace from the seed phrase. This adds the crucial step of using the trimmed seed phrase in the `onSubmit` handler.

### CODE REVIEW

**Review by danjm — APPROVED** [review]

File: ui/app/components/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js [hunk 1]
```diff
@@ -45,9 +45,11 @@ export default class ImportWithSeedPhrase extends PureComponent {
       } else if (!validateMnemonic(parsedSeedPhrase)) {
         seedPhraseError = this.context.t('invalidSeedPhrase')
       }
+
```
Comment (danjm) [inline ui/app/components/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js]: Test failures are real failures. `parsedSeedPhrase` will always remove trailing white spaces, not allowing the user to type spaced words.

**Review by danjm — CHANGES_REQUESTED** [review]
left comment

File: ui/app/components/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js (hunk 1 above)
Comment (whymarrh) [reply ui/app/components/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js]: Yeah totally didn't test this

**Comment by whymarrh** [comment]
Updated. I think the e2e failure is unrelated.

**Comment by kumavis** [comment]
@whymarrh looks like conflicts popped up again

**Review by danjm — APPROVED** [review]

**Pull request merged**